src/main/java/org/shanerx/tradeshop/TradeShop.java

```diff
@@ -43,6 +46,8 @@
 	private File settingsFile = new File(this.getDataFolder(), "config.yml");
 	private FileConfiguration settings;
 	
+	private ArrayList<Material> inventories = new ArrayList<>();
```

Comment:
Maybe it would be better to call it "shops" rather than "inventories" as it is not a List<Inventory>. Just a suggestion, though.